source/azure/activity-services/prerequisites/credentials.rst

```diff
@@ -113,70 +107,4 @@ Regardless of the service or activity to be monitored, the authentication file i
 
 Check the :doc:`azure-logs wodle </user-manual/reference/ossec-conf/wodle-azure-logs>` section from the ossec.conf reference page for more information about the ``<auth_path>`` and other available parameters.
 
-
-Inserting the credentials into the configuration
```

Comment:
Same with this, we will need to remove it in 5.0.0, but we should stop showing examples before.